src/pages/patch/Metadata.tsx

```diff
@@ -53,6 +56,22 @@ export const Metadata: React.FC<Props> = ({ loading, patch, error }) => {
           </StyledLink>
         </P2>
       )}
+      {parameters !== undefined && parameters.length > 0 && (
+        <P2>
+          <StyledLink
+            data-cy="parameters-link"
+            onClick={() => setShowModal(true)}
+          >
+            Patch Parameters
+          </StyledLink>
+        </P2>
+      )}
+      <ParametersModal
+        dataCy="parameters-modal"
+        parameters={parameters}
+        visible={showModal}
+        closeModal={() => setShowModal(false)}
+      />
     </MetadataCard>
```

Comment:
I think this should be moved to its own component. Like we do for some of the other modals. Instead of embedding it in the Metadata card component.

Reply:
The state logic for the modal and the link that it shows by default should live in the ParmatersModal File.